data/SampleData.js

```diff
@@ -0,0 +1,230 @@
+export const contactsSampleData = [
+  {
+    id: "1",
+    firstName: "Chandler",
+    middleName: "Muriel",
+    lastName: "Bing",
+    gender: "male",
+    profilePicture: "649e91836af01f9b18155053.jpg",
```

Comment:
One super small suggestion:
I know that this is sample data, but maybe you could give your images more relevant file names.

Reply:
I made hard to guess file names and a file mapping inside the object on purpose because of security reasons. The images are not behind authentication currently and if you know the person name for example you could guess the persons image link and download it, even when you did not create the contact in your account later on (when I implement authentication). 

But thanks for the hint.